src/tree/box.js

```diff
@@ -1,8 +1,8 @@
 import card from '../card/card';
 
-export default function box(divBox, o, nodeSize, appendNodes, reRender) {
+export default function box(divBox, o, appendNodes, reRender) {
```

Comment:
We should rename o here as well. I copied this from an example but it is not really clear what it represents right now. Will think of something

Reply:
Agree. It probably is short for orientation. It contains the object we return from the `position` file. Had the same discussion with Tobias. But will figure out something eventually :)